jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/uploadResource.feature

```diff
@@ -1,75 +1,75 @@
 Feature: Upload a Resource
 
-Scenario: Upload a JVM Resource
+  Scenario: Upload a JVM Resource
 
     Given I logged in
     And I am in the Configuration tab
 
-    # create media
+      # create media
```

Comment:
Please align the comment with the code so it is consistent with the rest of the features

    # create  media
    And blah blah blah....

Reply:
doing it

Reply:
done